### ISSUE TITLE
TensorFlow.js: Make a prediction from 2D Data

### ISSUE THREAD

**Issue by armelyara** [tensorflow/tfjs#4242]
I follow a lab on Google Codelabs about predictions and at the lab end, I didn't get the same score about prediction. 
In fact, I trained the model directly on my chrome Browser and i got as result a underfiting for predictions. 

async function trainModel(model, inputs, labels){
    //Préparer le modèle pour l'entrainement
    model.compile({
        optimizer: tf.train.adam(),
        loss: tf.losses.meanSquaredError,
        metrics: ['mse'],

    });
    const batchSize = 32;
    const epochs = 50;

    return await model.fit(inputs,labels, {
        batchSize,
        epochs,
        shuffle: true,
        callbacks: tfvis.show.fitCallbacks(
            {name: 'Training Performance'},
            ['loss', 'mse'],
            {height: 200, callbacks: ['onEpochEnd']}
        )
    });
}
I want to know if the bug is come from my browser ?

**Comment by tafsiri** [comment]
You generally might not get exactly the same values as the codelab due to a number of random variables in the models initialization, and varying behaviour in the hardware you are using. But as long as your loss is decreasing and your accuracy increases to a reasonable place then there is no bug to worry about.

**Comment by armelyara** [comment]
Ok thank you. I have other questions please.
Before congratulation for this lab.
Is it possible to get your contacts? I sent you relation quest on LinkedIn.

Best regards

**Comment by tafsiri** [comment]
Are your questions about TensorFlow.js? We/I can't offer general ml support unfortunately. This is a good place to file bugs with the library, but for general assistance we recommend checking out resources like https://www.tensorflow.org/resources/learn-ml/basics-of-tensorflow-for-js-development or asking on [stackoverflow](https://stackoverflow.com/questions/tagged/tensorflow.js) for technical questions.

**Comment by armelyara** [comment]
Ho ok i understand. Thank you Yannick.

**Comment by deveshsinghal22** [comment]
I am also trying the same codelab tuorial. 
below image is the result for Training Performance 
![image](https://user-images.githubusercontent.com/2578290/124344349-2dfaa600-dbef-11eb-99eb-450304774d83.png)


below image is what shows in google codelab
![image](https://user-images.githubusercontent.com/2578290/124344385-669a7f80-dbef-11eb-832c-56be3d15e243.png)


kindly confirm if this is ok. 

below are versions

"@tensorflow/tfjs": "^3.7.0",
 "@tensorflow/tfjs-vis": "^1.5.1",
